src/api/client.test.jsx

```diff
@@ -18,12 +18,13 @@ describe('API client requestAssets', () => {
     requestAssets(COURSE_ID, {
       page: 2,
       pageSize: 10,
-      sort: 'asc',
+      sort: 'content_type',
```

Comment:
Just fixing this up, since the API expects sort to be the name of the attribute on which to sort on and direction to be the direction ('asc' or 'desc').